flake.nix

```diff
@@ -0,0 +1,57 @@
+{
+  description = "soxin: opiniated configs for everyone";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    home-manager = {
+      url = "github:nix-community/home-manager";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    futils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, home-manager, futils } @ inputs:
+    let
+      inherit (nixpkgs) lib;
+      inherit (lib) recursiveUpdate;
+      inherit (futils.lib) eachDefaultSystem;
+
+      pkgImport = pkgs: system:
+        import pkgs {
+          inherit system;
+        };
+
+      multiSystemOutputs = eachDefaultSystem (system:
+        let
+          pkgs = pkgImport nixpkgs system;
+        in
```

Comment:
```suggestion
      multiSystemOutputs = eachDefaultSystem (system:
        let
          pkgs = import nixpkgs { inherit system; };
        in
```

Reply:
I think that since `nixpkgs` is only used here we can do something like `nixpkgs.${system}.legacyPackages`